spring-cloud-sleuth-zipkin/src/test/java/org/springframework/cloud/sleuth/zipkin/HttpZipkinSpanReporterTest.java

```diff
@@ -68,10 +69,9 @@ public void postsSpans() throws Exception {
 
 	@Test
 	public void postsCompressedSpans() throws Exception {
-		this.reporter = new HttpZipkinSpanReporter(
-				this.zipkin.httpUrl(),
+		this.reporter = new HttpZipkinSpanReporter(restTemplateWithCompression(), this.zipkin.httpUrl(),
 				0, // so that tests can drive flushing explicitly
-				false, // enable compression
+				true,
```

Comment:
not necessary if the reader is expected to know that this parameter index
controls compression.. basically the comment was for the reader, not the
writer. If I didn't comment this, you might haven't as easily noticed a bug
later, right?